File: python/quotes.py
from Main import Controller
import re

def main():
    args = Controller.arg_parse()
    t = Controller.read_file(args.t)
    regex = r"((?<![\\])['\"])((?:.(?!(?<![\\])\1))*.?)\1"	
    matches = re.findall(regex,t,flags=re.I)
    for delimit,match in matches: print match

if __name__ == "__main__":
    main()

File: julia/quotes.jl
#!/usr/bin/julia

include("Main.jl")
using Controller

function main()
    args = Controller.arg_parse()
    t = Controller.read_file(args["t"])
    t = """ $t """
    pattern = r"((?<![\\])['\"])((?:.(?!(?<![\\])\1))*.?)\1"
    result = [m for m in matchall(pattern,t)]
    for i = 1:len(result)
        println(result[i])
    end
end

main()